integration/test_qca.in

```diff
@@ -0,0 +1,6 @@
+(FIND-QCA-PATH : TARGET ("SHC1") : SOURCE ("IRS1") : RELTYPE ("controls-state-change-of"))
```

Comment:
In the message format I think the name of an argument has to follow the : directly without a space in between, for instance
```(FIND-QCA-PATH :TARGET ("SHC1") :SOURCE ("IRS1") :RELTYPE ("controls-state-change-of"))```